easybuild/easyblocks/e/easybuildmeta.py

```diff
@@ -69,14 +69,16 @@ def install_step(self):
 
         try:
             subdirs = os.listdir(self.builddir)
-            for pkg in ['easyconfigs', 'easyblocks', 'framework']:
-                seldirs = [x for x in subdirs if x.startswith('easybuild-%s-' % pkg)]
+            for pkg in ['vsc-base', 'easybuild-framework', 'easybuild-easyblocks', 'easybuild-easyconfigs']:
+                seldirs = [x for x in subdirs if x.startswith(pkg)]
                 if not len(seldirs) == 1:
-                    self.log.error("Failed to find EasyBuild %s package (subdirs: %s, seldirs: %s)" % (pkg, subdirs, seldirs))
-
-                self.log.debug("Installing EasyBuild package %s" % pkg)
-                os.chdir(os.path.join(self.builddir, seldirs[0]))
-                super(EB_EasyBuildMeta, self).install_step()
+                    if pkg != 'vsc-base':
```

Comment:
Note that this is handled better in the current version of the `EasyBuildMeta` easyblock.

This patch is just to release EB v1.16.2, which is required to install EasyBuild v2.0.0 'offline' using EasyBuild (EB v1.16.1 doesn't work).